PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/FullGfaReader.java

```diff
@@ -77,34 +77,41 @@ private void read() {
     System.out.println("Number of lines read: " + count);
   }
 
+  /**
+   * Private method for handling an "H" as first character on the line of a gfa file.
+   * @param sc The scanner which is used to parse the file.
+   */
   private void handleH(Scanner sc) {
     String line = sc.nextLine();
-    this.genoms = extractGenoms(line);
+    this.genoms = extractGenomes(line);
 
   }
 
-  private ArrayList<String> extractGenoms(String line) {
-    String[] words = line.split(":");
-    if (words[0].endsWith("ORI")) {
-      String[] gens = words[2].split(";");
-      ArrayList<String> gensAr = new ArrayList<>();
-      gensAr.addAll(Arrays.asList(gens));
-      return gensAr;
-    }
-    return null;
-  }
-
+  /**
+   * Private method for handling an "S" as first character on the line of a gfa file.
+   * @param sc The scanner which is used to parse the file.
+   */
   private void handleS(Scanner sc) {
     int nodeId = sc.nextInt();
     String bases = sc.next();
     String line = sc.nextLine();
     String[] words = line.split("\\s+");
-    ArrayList<String> nodeGens = extractGenoms(words[2]);
+    ArrayList<String> nodeGens = extractGenomes(words[2]);
+    int orientation = extractStart(words[6]);
+    //words[3] = CRD:Z:XXXX
+    //words[4] = CRDCTG:Z:XXXX
+    //words[5] = CTG:Z:XXXX;XXXX;...
+    //words[6] = START:Z:INT
     Node no = originalGraph.getNode(nodeId);
     no.setGenomes(nodeGens);
     no.setBases(bases);
+    no.setAlignment(orientation);
```

Comment:
@cbilstra I rather it is changed now. It is a minor change and will probably be forgotten. We'll see it as 'refactoring the code somewhat and testing it' rather than only 'testing it'. A.k.a. Cleanup